<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-FLJMP0F" failures="0" tests="14" name="com.demo.testcases.TC_MISC_WorkingFlow_001" time="48185.543" errors="1" timestamp="2021-03-09T10:10:16 IST" skipped="0">
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="afterSuite" time="0.036">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentReports.flush()&quot; because &quot;com.demo.utility.ExtentManager.extent&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.flush()" because "com.demo.utility.ExtentManager.extent" is null
at com.demo.utility.ExtentManager.endReport(ExtentManager.java:37)
at com.demo.Base.base.afterSuite(base.java:199)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:351)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- afterSuite -->
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="clickonLoginTest" time="220.640"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="navigate_1099MISCform" time="240.338"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="form_addpayer" time="576.016"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="form_addRecipient" time="46262.109"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="singleBox" time="0.129"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="saveForm1099" time="81.236"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="validateVES_data" time="479.040"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="launch_paymentPage" time="133.402"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="paymentpage_validations" time="60.922"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="payment_confirmationPage" time="13.756"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="filing_history" time="13.514"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="manageforms" time="4.095"/>
  <system-out/>
  <testcase classname="com.demo.testcases.TC_MISC_WorkingFlow_001" name="dashboardsearch_submitted" time="100.313"/>
  <system-out/>
</testsuite> <!-- com.demo.testcases.TC_MISC_WorkingFlow_001 -->
